src/components/Cardgrid.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  CircularProgress,
+  FontIcon,
+  IconButton,
+} from 'material-ui';
+import {
+  GridList,
+  GridTile
+} from 'material-ui/GridList';
+import { muiThemeable } from 'material-ui/styles';
+
+const styles = {
+  image: {
+    width: 'initial',
+    height: 'initial',
+    maxWidth: '100%',
+    maxHeight: '100%',
+  },
+  load: {
+    width: '100%',
+    height: 135,
+    paddingTop: 40,
+    boxSizing: 'border-box',
+    backgroundColor: '#606060',
+    textAlign: 'center',
+  },
+  loadmore: {
+    color: 'white',
+    fontSize: 30,
+    cursor: 'pointer',
+    textDecoration: 'underline',
+    backgroundColor: 'transparent',
+    border: 'none',
+    outline: 'none',
+  }
+};
```

Comment:
I found [presentation](https://oliviertassinari.github.io/a-journey-toward-better-style/) of the author of `material-ui` about props and cons of using inline styles in `javascript`. You can found discussion in [repo](https://github.com/oliviertassinari/a-journey-toward-better-style)  of the author and on `material-ui` [website](http://www.material-ui.com/#/customization/styles).

Cons for me:

- it's hard to maintain a lot of styling code in javascript (it's simpler with eg. SASS)
- I can't separate `style` layer from `javascript`
- I suppose that some features from CSS are harder (or impossible) to achieve in inline styling
- Today I tried to override style of `placeholder` in `input` but it's appeared that in `material-ui` `placeholder` is not a real `input-placeholder` but `div` with special styling. I'm afraid that there would be more surprises like this.